### ISSUE TITLE
Problem Text at top of problem page should match current user's version, if available

### ISSUE THREAD

**Issue by boonebgorges** [livinglab/webwork-for-wordpress#13]
Flow will look like this:

- If a user is logged in, and has already posted a question on WWWP for this specific problem while coming from WW, then we should have that user's version of the problem. Show it.
- If a user is logged in, and has just come directly from WW, then we should have the version sent via POST (available via post_data_key). Show it.
- Otherwise, fall back on the "canonical" version of the problem text, which is the version belonging to the person who first posted a question about a given problem.

**Comment by moui72** [comment]
is this ready to test?

**Comment by boonebgorges** [comment]
No. Due to time constraints, and the fact that this is a fairly low-impact issue, it will probably have to be pushed from the milestone.

**Comment by moui72** [comment]
k, just checking

**Comment by boonebgorges** [comment]
We already do the first and third items described above, so I think we just need to do the second. That feels very low-priority, though, because it'll only happen in cases where someone who's already asked a question about a specific problem goes to ask *another* question about it.
